src/constants/dexes.ts

```diff
@@ -272,7 +274,17 @@ export const dexListConfig: DexList = {
     name: 'Swapr',
     icon: 'https://swapr.eth.link/favicon.png',
     chainIds: [ChainId.ARBITRUM]
-  }
+  },
+  wagyuswap: {
+    name: 'WagyuSwap',
+    icon: 'https://exchange.wagyuswap.app/images/logo.png',
+    chainIds: [ChainId.VELAS]
+  },
+  astroswap: {
+    name: 'AstroSwap',
+    icon: 'https://exchange.astroswap.app/images/logo.png',
+    chainIds: [ChainId.VELAS]
+  },
```

Comment:
remember update these information, 2 new dexes and 1 new chain
<img width="507" alt="image" src="https://user-images.githubusercontent.com/11427213/156482085-a777d37a-8c20-42f1-b255-309d0d831a7a.png">